Add HTTP Basic Auth (#7)
juanmanuelramallo/dokkunductor: README.md

```bash
sudo dokku plugin:install https://github.com/dokku/dokku-postgres.git postgres
```

## 4. HTTP Basic authentication

> Set the `HTTP_BASIC_AUTH_NAME` and `HTTP_BASIC_AUTH_PASSWORD` in your `.env` file to whatever you want.

Reviewer: ```suggestion
Set the `HTTP_BASIC_AUTH_NAME` and `HTTP_BASIC_AUTH_PASSWORD` to configure basic authentication.

```bash
dokku config:set dokkunductor HTTP_BASIC_AUTH_NAME=??? HTTP_BASIC_AUTH_PASSWORD=???
```
```
